### PR TITLE
Add page on governance

### PR DESCRIPTION
**Pull request by petrasovaa**
image

### CODE REVIEW

File: content/about/governance.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Governance"
+date: 2024-08-27T10:12:05+02:00
+layout: "general"
+---
+
+# Governance
+The Project Steering Committee guides the GRASS project’s direction, budget, and
+governance, making decisions through voting and overseeing the Developer Team
```
Comment (neteler) [inline content/about/governance.md:9]:
```suggestion
governance, making decisions through voting and overseeing the Development Team
```

I think we commonly call it "GRASS Development Team".

File: content/about/governance.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+which manages source code changes. The project operates under the support of the
```
Comment (neteler) [inline content/about/governance.md:10]:
```suggestion
which manages source code changes. The project operates under the auspices of the
```

File: content/about/governance.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Open Source Geospatial Foundation (OSGeo).
+
+## Guided by a Project Steering Committee
+
+<a href="/images/other/grass_psc_team_prague_2024.jpg">
+  <img src="/images/other/grass_psc_team_prague_2024.jpg"
+   alt="PSC members in Prague 2024 Community Meeting"
+   title="PSC members in Prague 2024 Community Meeting"
```
Comment (neteler) [inline content/about/governance.md:18]:
```suggestion
   alt="PSC members at the Prague 2024 Community Meeting"
   title="PSC members at the Prague 2024 Community Meeting"
```

File: content/about/governance.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+processes. More specifically, the PSC votes on write access to the source code
+repositories, reimbursements, and other issues that are difficult to decide by
+developers’ consensus. Major technological or procedural changes are first
+proposed and discussed in the form of Request for Comments (RFC) documents and
+later voted on by the PSC. The PSC has 9 members and is elected by the
+contributor community.
+
+The PSC has a Chair selected from and by the <a href="/about/team">PSC
+members</a>. The PSC may create additional roles such as a treasurer and appoint
+people to these roles. The PSC appoints and oversees the Developer Team and
```
Comment (neteler) [inline content/about/governance.md:33]:
```suggestion
people to these roles. The PSC appoints and oversees the Development Team and
```

Comment (neteler) [reply content/about/governance.md:33]: Also to be changed below, accordingly.

File: content/about/governance.md (hunk 1 above)
Comment (wenzeslaus) [reply content/about/governance.md:9]: Good catch. By the way, how would you define it?

Comment (neteler) [reply content/about/governance.md:9]: Suggestion:

The GRASS Development Team is an international group of developers and scientists from various fields who work together to continuously improve the capabilities of GRASS GIS. Its diverse composition allows a wide range of expertise to be applied to the development of the software.

File: content/about/governance.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+Development Team and a broader consensus is sought out for larger or high impact
+change requests. Anyone, including the original change request contributor, may
+participate in the discussion about suitability of the proposed code changes or
+provide reviews. However, at least one formal approving review by a Development
+Team member is required for inclusion into the source code. The same rules apply
+when a Development Team member creates a change request, i.e., another member
+needs to perform the review and approve the change.
+
+Development Team members may involve or may defer a decision to the PSC or its
+individual members where consensus cannot be reached or when circumstances seem
```
Comment (veroandreo) [inline content/about/governance.md:90]:
```suggestion
Core Development Team members may involve or may defer a decision to the PSC or its
individual members when consensus cannot be reached or when circumstances seem
```

**Review by veroandreo — APPROVED** [review]
After some discussion on how to define "the team", we came up with "core" to actually indicate and acknowledge that there's a group of devs that are actually managing the changes and maintaining the source code, among other things.

File: content/about/governance.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+date: 2024-08-27T10:12:05+02:00
+layout: "governance"
+---
+
+The Project Steering Committee guides the GRASS project’s direction, budget, and
+governance, making decisions through voting and overseeing the GRASS Development
+Team which manages source code changes. The project operates under the auspices
+of the Open Source Geospatial Foundation (OSGeo).
+
+## Guided by a Project Steering Committee
```
Comment (wenzeslaus) [inline content/about/governance.md:12]:
```suggestion
## Guided by the Project Steering Committee
```

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

**Review by veroandreo — APPROVED** [review]

**Review by wenzeslaus — APPROVED** [review]

**Pull request merged**